apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -267,6 +302,11 @@ const TicketIdPage = () => {
                                         </Col>
                                         <Col span={24}>
                                             <Typography.Text style={{ fontSize: '24px' }}>{ticket.details}</Typography.Text>
+                                            { 
+                                                !isEmpty(files) 
+                                                    ?
+                                                    <TicketFileList files={files} /> : null
+                                            }
```

Comment:
It seems to me, that this is shorter:
```suggestion
                                            {!isEmpty(files) && (
                                                    <TicketFileList files={files} />
                                            )}
```

Reply:
Done